### PR TITLE
Update to logseq extension 

### PR DESCRIPTION
**Pull request by sawhney17**
## Description

Adds another command to the logseq extension called Logseq Search (LogSearch). This allows you to search your logseq database using either fuzzy search or simple keyword matched search. 

When you find the page you want to open, just hit enter, and it will open in your current logseq window or a new one if the desired database is not the one currently open in logseq. 

This uses the logseq URL scheme in the latest version of logseq. 

## Screencast
Untitled Project 1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pomdtr** [comment]
I think that setting the full path as a subtitle is a bit noisy. What do you think of using the filename instead ?

**Comment by sawhney17** [comment]
Sounds like a good idea. Just pushed a change.

**Comment by pernielsentikaer** [comment]
Hi @sawhney17 👋

Thanks for your contribution 🔥 

@futantan do you have time to give it a quick spin?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @sawhney17 👋 

I think I would change so `showGraphPathInvalidToast` doesn't feel that "heavy" to read by splitting the sentence, maybe like this:

```
export const showGraphPathInvalidToast = () => {
  showToast({
    style: Toast.Style.Failure,
    title: "Logseq graph path is invalid",
    message: "Update the path and try again.",
  });
};
```

I also noticed that when I wrong path is set and I'm trying to submit a `Quick Note` it flashes ["Logseq graph path is invalid" toast](https://github.com/sawhney17/extensions/blob/main/extensions/logseq/src/index.tsx#L26) for a few milliseconds before show [the one from the catch](https://github.com/sawhney17/extensions/blob/main/extensions/logseq/src/index.tsx#L34). Can you look into this?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by futantan** [comment]
> Hi @sawhney17 👋
> 
> Thanks for your contribution 🔥
> 
> @futantan do you have time to give it a quick spin?

Too busy recently, but I will ASAP

**Comment by pernielsentikaer** [comment]
Hi @sawhney17 👋 

Do you have an update? Do you need any help?

**Comment by dsumin** [comment]
Thank you for a great extension!

I'm not sure if the issue is caused by Logseq setup, but without appending `.md` file extension [here](https://github.com/raycast/extensions/pull/1484/files?short_path=8979529#diff-db9b18b5e049097fae7fb141d31d15151cae4d114e7dfa0aa7e01e7d9b6c2a24R45) Loqseq fails to open any page from search results:
`            <Action.OpenInBrowser title="Open in Logseq" url={searchResult.url + ".md"} />`

**Comment by pernielsentikaer** [comment]
@sawhney17 are you still around?

**Comment by sawhney17** [comment]
Hey Sorry, for the delay, just pushed the fixes mentioned in the thread. I’m unable to replicate the issue with the filepath being invalid.

**Comment by pernielsentikaer** [comment]
Hi @sawhney17 👋 

Thanks for taking your time to look into corrections 🔥 

@pomdtr Are you by any chance using logseq and could give it a spin? 
@futantan do you have time to check?
@dsumin can you give it an extra spin too?

Thanks 🔥

**Comment by dsumin** [comment]
> @dsumin can you give it an extra spin too?
Works fine now, @sawhney17  thank you! 🔥

**Comment by sawhney17** [comment]
@pernielsentikaer 

How close are we to getting this update merged? Looking forward to getting the store version so that'd it'd be easy to use on multiple machines 😅

**Comment by pernielsentikaer** [comment]
We're waiting on approval from @futantan - do you have time to check?

**Comment by futantan** [comment]
I will check it today and let you know

File: extensions/logseq/src/logsearch.tsx
Comment (futantan) [inline extensions/logseq/src/logsearch.tsx:16]: It's better to wrap `validateUserConfigGraphPath` in `useEffect`

**Comment by pernielsentikaer** [comment]
@sawhney17 can you look into the feedback from @futantan

**Comment by sawhney17** [comment]
@pernielsentikaer, should be ready to go now

**Comment by pernielsentikaer** [comment]
Let's get it merged and get some feedback 🔥 

Thanks for your time @sawhney17 and @futantan for the feedback

**Comment by absir1949** [comment]
@sawhney17 thanks for your contribution.

i try it and i find it is not Support Chinese search. do  it will support Chinese search?

thank you

**Comment by sawhney17** [comment]
Could you try with the smart search setting off? I'll have to look into this.

**Pull request merged**
